js_modules/dagster-ui/packages/ui-core/src/assets/AssetView.tsx

```diff
@@ -177,7 +177,7 @@ export const AssetView = ({
   };
 
   const renderPartitionsTab = () => {
-    if (definition?.isSource) {
+    if (!definition?.isMaterializable) {
```

Comment:
Same note as above, I think we want isMaterializable = false specifically. Maybe another way of writing this is `definition?.isMaterializable === false`, since the other falsy value is undefined

Reply:
Oo thanks for catching my ignoramus javascript, will fix